ios/Plugin/PrivacyScreenPlugin.swift

```diff
@@ -36,9 +41,20 @@ public class PrivacyScreenPlugin: CAPPlugin {
         guard self.isEnabled else {
             return
         }
+
         self.privacyViewController = UIViewController()
-        self.privacyViewController!.view.backgroundColor = UIColor.gray
+        self.privacyViewController!.view.backgroundColor = .clear;
         self.privacyViewController!.modalPresentationStyle = UIModalPresentationStyle.overFullScreen
+        if !UIAccessibility.isReduceTransparencyEnabled {
+            let blurEffect = UIBlurEffect(style: .dark)
+            let blurEffectView = UIVisualEffectView(effect: blurEffect)
+            blurEffectView.frame = self.privacyViewController!.view.bounds
+            blurEffectView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+            self.privacyViewController!.view.addSubview(blurEffectView)
+        } else {
+            self.privacyViewController!.view.backgroundColor = .gray
+        }
```

Comment:
Please make sure that you do not submit multiple features in one PR. This PR should only contain changes that are actually needed for the feature.
Are these changes necessary for detection? If yes, why?

Reply:
Nope. It is another feature to make blur effect instead of grey background. Last time we decided to make config so user could pass the color of background or blured.

Can i make one method for ex. ```setConfigs``` in frontend, where user could pass options?

Reply:
If ok can you define method as it should be and i will continue

Reply:
Let's discuss this in the other PR (#34), please. I will be happy to make a suggestion for the interface. In this PR the code should not be included.